objcgen/embedder.cs

```diff
@@ -279,6 +281,8 @@ public int Compile ()
 
 			BuildInfo[] build_infos;
 
+			string bitcodeFlags = NoBitCode ? null : "-fembed-bitcode";
```

Comment:
that should move into the `switch(platform)`
so iOS can remain *bitcode free* by default
while the tvOS and watchOS continue to have bitcode enabled by default